resources/assets/coffee/react/beatmap-discussions/post.coffee

```diff
@@ -290,6 +290,16 @@ class BeatmapDiscussions.Post extends React.PureComponent
 
   permalink: (e) =>
     e.preventDefault()
+    targetEl = e.currentTarget
+
+    # copy url to clipboard
+    clipboard.writeText targetEl.href
+
+    # show feedback
+    currentLabel = targetEl.innerHTML
+    targetEl.innerHTML = osu.trans('common.buttons.permalink_copied')
+    Timeout.set 2000, ->
+      targetEl.innerHTML = currentLabel
```

Comment:
this looks like it'll break when navigating away... except it doesn't ಠ_ಠ

Reply:
🤷‍♀️